adapter/internal/adapter/adapter.go

```diff
@@ -312,4 +312,7 @@ func fetchAPIsOnStartUp(conf *config.Config) {
 			}(data)
 		}
 	}
+	// All apis are fected. Deploy the /ready route for the readiness and startup probes.
+	xds.DeployReadinessAPI(envs)
+	logger.LoggerMgw.Info("Fetching APIs completed...")
```

Comment:
Its better if we can print the number of APIs fetched as well

Reply:
```
2021-06-27 17:32:38 INFO [apis_fetcher.go:249] - [synchronizer.PushAPIProjects] [-] Successfully deployed 3 APIs
2021-06-27 17:32:38 INFO [server.go:224] - [xds.DeployReadinessAPI] [-] Finished fetching APIs from the Control Plane. Deploying the readiness endpoint...
```

Reply:
Fixed in https://github.com/wso2/product-microgateway/pull/2101/commits/f8b1a0f26d76982947e760b92ce30641bbf7e936